schematools/utils.py

```diff
@@ -18,10 +18,19 @@
 
 
 @ttl_cache(ttl=16)  # type: ignore
-def schema_defs_from_url(schemas_url) -> Dict[str, types.DatasetSchema]:
-    """Fetch all schema definitions from a remote file.
+def schema_defs_from_url(
```

Comment:
Het lijkt wel alsof de regellengte waar Black mee werkt een stuk minder is dan 99.

Reply:
Uhm, vreemd, ik heb opnieuw expliciet opgegeven dat Black naar 99 regellengte moet kijken. Dit werkt. Nu wordt er niet te vroeg afgekapt...